syntax = "proto3";
package auth;

import "google/protobuf/empty.proto";

message Token {
    int32 user_type = 1;
    string session_id = 2;
}

message AuthResponse {
    int32 code = 1;
    string detail = 2;
  }

  service AuthService {
    rpc AuthUser(Token) returns (AuthResponse) {}
  }